lib/apiUrl.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+	'use strict';
+
+	var _ = require('lodash'),
+		qs = require('qs'),
+		protocol = 'https',
+		host = process.env.PIPEDRIVE_API_HOST || 'api.pipedrive.com',
+		version = process.env.PIPEDRIVE_API_VERSION || 'v1',
+		baseUri = protocol + '://' + host + '/' + version;
+
+	module.exports = function apiUrl(path, options, tokenNeeded) {
+		var queryObj = {};
+		if (!!tokenNeeded) {
```

Comment:
no need to boolean it, if it evaluates to false initially then it will be false by the if too